src/fastlane/release_actions/fastlane-plugin-release_actions.gemspec

```diff
@@ -0,0 +1,35 @@
+# coding: utf-8
+
+lib = File.expand_path("../lib", __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'fastlane/plugin/release_actions/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = 'fastlane-plugin-release_actions'
```

Comment:
This is the way the generator builds plugins. I think to be safe, we should stick to their format (because i don't know anything about fastlane internals, but I can make some guesses).